### DISCUSSION TITLE
Storybook 7 for Vue - detect theme change event

### DISCUSSION THREAD

**Discussion by dakt** [storybookjs/storybook#24308] [Help]
How can I detect theme change?

**Comment by jdassin** [comment]
Hello @dakt , if you encounter any concerns or need assistance with the Storybook & Vue library, please check out https://github.com/storybook-vue. It's primarily dedicated to Vue and managed by the fantastic @chakAs3, who is also a Storybook maintainer and will surely assist you promptly.
